### PR TITLE
call utilflag.InitFlags all over this repo

### PR DESCRIPTION
**Pull request by hzxuzhonghu**
**What this PR does / why we need it**:

follow up #59718 ,
we switch everything over to Cobra commands, we can go back to calling	`utilflag.InitFlags()`
// utilflag.InitFlags() (by removing its pflag.Parse() call)

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by deads2k** [comment]
/assign @ncdc 
As I recall, this was your comment originally.

**Comment by liggitt** [comment]
need to make sure this works with all the ways we invoke things (directly, via hyperkube, etc). similar changes have broken things (c.f. https://github.com/kubernetes/kubernetes/pull/60379)

**Comment by ncdc** [comment]
@liggitt yeah, #57672 should never have gone in by itself. This change needs to be all-or-nothing.

File: test/images/no-snat-test-proxy/main.go [hunk 1]
```diff
@@ -40,20 +43,36 @@ func NewMasqTestProxy() *MasqTestProxy {
...
+	m := NewMasqTestProxy()
+	cmd := &cobra.Command{
+		Use:  "MasqTestProxy",
+		Long: `MasqTestProxy`,
+		Run: func(cmd *cobra.Command, args []string) {
+			verflag.PrintAndExitIfRequested()
+			m.Run()
+		},
+	}
+	m.AddFlags(pflag.CommandLine)
```
Comment (ncdc) [inline test/images/no-snat-test-proxy/main.go]: @liggitt this should be `cmd.Flags()`, right?

File: test/images/no-snat-test/main.go [hunk 2]
```diff
@@ -48,16 +52,32 @@ func (m *MasqTester) AddFlags(fs *pflag.FlagSet) {
...
 	m := NewMasqTester()
+	cmd := &cobra.Command{
+		Use:  "MasqTest",
+		Long: `MasqTest`,
+		Run: func(cmd *cobra.Command, args []string) {
+			verflag.PrintAndExitIfRequested()
+			m.Run()
+		},
+	}
 	m.AddFlags(pflag.CommandLine)
```
Comment (ncdc) [inline test/images/no-snat-test/main.go]: @liggitt this should be `cmd.Flags()`, right?

File: test/images/no-snat-test-proxy/main.go (hunk 1 above)
Comment (liggitt) [reply test/images/no-snat-test-proxy/main.go]: yes. in general, we shouldn't be moving toward global flagsets anywhere

Comment (hzxuzhonghu) [reply test/images/no-snat-test-proxy/main.go]: @hanxiaoshuai I will update and squash into your commit.

**Comment by hzxuzhonghu** [comment]
@ncdc @liggitt comments addressed, after this all components use `utilflag.InitFlags()`

**Comment by ncdc** [comment]
I recommend this not go into v1.10 and instead go in as soon as master opens for v1.11 development.

**Comment by hzxuzhonghu** [comment]
@ncdc Now comes at 1.11, could we move forward now?

**Comment by ncdc** [comment]
SGTM

**Comment by ncdc** [comment]
LGTM. Would appreciate another set of 👀 on it.

**Pull request closed without merging**